pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java

```diff
@@ -68,12 +85,17 @@ public String formatModule(Module module) throws Exception {
 			VcsStatus status = module.getVcs().getSupport().getStatus(moduleDirectory);
 			String branch = status.getBranch();
 			StringBuilder line = new StringBuilder();
+
 			line.append(status.hasUnpublishedChanges() ? 'M' : ' ');
 			line.append(status.hasUncommittedChanges() ? 'M' : ' ');
 			line.append(' ').append(module.getName());
-			line.append(' ');
+
+			line.append(new String(new char[moduleMaxLength + 2 - module.getName().length()]).replace("\0", " "));
```

Comment:
Whoa, this is an original idea. :) But not very efficient. Please use `Strings.repeat()` instead.